src/udp_wrap.cc

```diff
@@ -175,6 +175,19 @@ void UDPWrap::GetFD(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(fd);
 }
 
+int uv_sockaddr_for_family(int address_family,
```

Comment:
Using a `uv_*` prefix is a bit confusing as it implies it's a libuv function. Maybe just `sockaddr_for_family`?

Reply:
Good point, I'll rename it. Thanks!